src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.cs

```diff
@@ -111,54 +62,19 @@ internal class CacheEntry : ICacheEntry
         /// </summary>
         public TimeSpan? SlidingExpiration
         {
-            get
-            {
-                return _slidingExpiration;
-            }
-            set
-            {
-                if (value <= TimeSpan.Zero)
-                {
-                    throw new ArgumentOutOfRangeException(
-                        nameof(SlidingExpiration),
-                        value,
-                        "The sliding expiration value must be positive.");
-                }
-                _slidingExpiration = value;
-            }
+            get => _slidingExpiration;
+            set => _slidingExpiration =  !value.HasValue || value > TimeSpan.Zero ? value : throw new ArgumentOutOfRangeException(nameof(SlidingExpiration), value, "The sliding expiration value must be positive.");
```

Comment:
I'm not sure this is "better" than what was previously there. IMO it is a lot harder to interpret what is happening.

Is this simply a "style" refactoring? Or does this actually have perf impact?

Reply:
good point, reverted